slack-gamebot/commands/leaderboard.rb

```diff
@@ -10,11 +11,12 @@ def self.call(client, data, match)
           max = nil
         else
           max = Integer(arguments.first)
-        end if arguments.any?
+        end if arguments.first
+        reverse = true if arguments.include? 'esrever' # This is a hidden magic trick which will reverse the ranks
         ranked_players = client.owner.users.ranked
         if ranked_players.any?
-          message = ranked_players.asc(:rank).limit(max).map do |user|
-            "#{user.rank}. #{user}"
+          message = ranked_players.send(reverse ? :desc : :asc, :rank).limit(max).each_with_index.map do |user, index|
+            "#{index + 1}. #{user}"
```

Comment:
I think we want the user rank here, especially in the non reverse case. That's because you can have users with the same rank I believe.